src/Infrastructure.VS.UnitTests/SolutionInfoProviderTests.cs

```diff
@@ -49,10 +49,23 @@ public void CheckIsSharedMefComponent()
             MefTestHelpers.CheckIsSingletonMefComponent<SolutionInfoProvider>();
         }
 
+        [TestMethod]
+        public void Ctor_DoesNotCallServices()
+        {
+            // The MEF constructor needs to be free-threaded, so it shouldn't switch threads or
+            // call any components that switch threads.
+            // In this case, we're not expecting it to call anything.
+
+            var serviceProvider = new Mock<IServiceProvider>();
+            _ = CreateTestSubject(serviceProvider.Object);
+
+            serviceProvider.Invocations.Should().BeEmpty();
+        }
+
         [TestMethod]
         [DataRow(null)]
         [DataRow("a value")]
-        public async Task Get_ReturnsExpectedValue(string solutionNameToReturn)
+        public async Task GetAsync_ReturnsExpectedValue(string solutionNameToReturn)
```

Comment:
I renamed the existing tests `GetAsync_xxx` since we now have `Get_xxx` versions.